cmd/openshift-newrelic-synthetics/sync/command.go

```diff
@@ -174,6 +176,8 @@ func Command(app *kingpin.Application) {
 	command.Flag("new-relic-api-key", "API key for authenticating with New Relic").Envar("NEW_RELIC_API_KEY").Required().StringVar(&c.NewRelicAPIKey)
 	command.Flag("new-relic-location", "Location which monitors will be provisioned").Default("AWS_AP_SOUTHEAST_2").StringVar(&c.NewRelicLocation)
 
+	command.Flag("monitor-type", "Type of New Relic Synthetics monitor to use").Default("SIMPLE").Envar("NEW_RELIC_MONITOR_TYPE").StringVar(&c.MonitorType)
```

Comment:
PING?

Reply:
SIMPLE in API == PING in the UI

Reply:
https://docs.newrelic.com/docs/apis/synthetics-rest-api/monitor-examples/manage-synthetics-monitors-rest-api/#api-names